packages/navigator/src/browser/navigator-contribution.ts

```diff
@@ -429,6 +443,22 @@ export class FileNavigatorContribution extends AbstractViewContribution<FileNavi
         this.tabbarToolbarRegistry.registerItem(item);
     };
 
+    /**
+     * Evaluates the widget that is triggered by the right click.
+     * @param event: `event` to be used when searching for the title and the tab-bar.
+     *
+     * @returns `widget` of the respective `title` if it exists, else returns undefined.
+     */
+    private getTargetedWidget(event?: Event): Widget | undefined {
+        let title: Title<Widget> | undefined;
```

Comment:
Why don't use `shell.findWidgetForElement` for event target?

Reply:
using `shell.findWidgetForElement` returns the entire tab-bar, so in order to still reveal the right file in the explorer we will have to go through the process of finding the title and returning the widget for that title, imo.